### PR TITLE
fix: Update Kata download URL

### LINKED ISSUE #1387: Kata URL has changed and needs to be updated in the code
**Is this a request for help?**:

Yes

---

**Is this an ISSUE or FEATURE REQUEST?** (choose one):
ISSUE
---

**What version of aks-engine?**:
0.34.3
---

**Kubernetes version**:
1.14

**What happened**:
Not able to use kata-container runtime using aks-engine.

**What you expected to happen**:
kata-container runtime should have been provisioned.

**How to reproduce it** (as minimally and precisely as possible):
Just create a cluster using aks-engine and in the ARM template:
        "containerRuntime": "kata-containers",

**Anything else we need to know**:

### PR DESCRIPTION
**Pull request by Jukie**
**Reason for Change**:

Fixes #1387

**Issue Fixed**:

Fixes #1387

**Requirements**:

- [ x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ n/a] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: parts/k8s/cloud-init/artifacts/cse_install.sh [hunk 1]
```diff
@@ -137,13 +137,15 @@ installMoby() {
 installKataContainersRuntime() {
     # TODO incorporate this into packer CI so that it is pre-baked into the VHD image
     echo "Adding Kata Containers repository key..."
+    ARCH=x86_64
```
Comment (CecileRobertMichon) [inline parts/k8s/cloud-init/artifacts/cse_install.sh]: can we do `ARCH=$(arch)` to make this more reusable?

File: parts/k8s/cloud-init/artifacts/cse_install.sh [hunk 2]
```diff
@@ -137,13 +137,15 @@ installMoby() {
...
+    BRANCH=stable-1.7
```
Comment (CecileRobertMichon) [inline parts/k8s/cloud-init/artifacts/cse_install.sh]: where does stable-1.7 come from?

**Comment by CecileRobertMichon** [comment]
@Jukie thanks for the PR!

Could you please `git commit --amend` when you rebase so the commit message has the `fix: ` prefix?

fix: Update Kata download URL

Otherwise our release tools won't include this in the CHANGELOG.

File: parts/k8s/cloud-init/artifacts/cse_install.sh (hunk 1 above)
Comment (Jukie) [reply parts/k8s/cloud-init/artifacts/cse_install.sh]: Good point, I'll update this now.

File: parts/k8s/cloud-init/artifacts/cse_install.sh (hunk 2 above)
Comment (Jukie) [reply parts/k8s/cloud-init/artifacts/cse_install.sh]: From http://download.opensuse.org/repositories/home:/katacontainers:/releases:/x86_64:/
Is there a better place to track the version in use? I assumed pinning to master isn't ideal

Comment (CecileRobertMichon) [reply parts/k8s/cloud-init/artifacts/cse_install.sh]: nope, just curious. I agree, let's not pin to master :)

**Comment by Jukie** [comment]
Sure, I had already amended it before your comment. Does it need to be lowercase as well?

**Comment by CecileRobertMichon** [comment]
> Sure, I had already amended it before your comment. Does it need to be lowercase as well?

Yeah, it does. It should be `fix: `

**Comment by Jukie** [comment]
Okay, please take another look when you can. Should be good to go

**Pull request merged**